File: src/dotnet/FaxCore.ApiSamples/Samples/Administration/User/ListUsers.cs
using System.Threading.Tasks;

namespace FaxCore.ApiSamples.Samples.Administration.User
{
    internal static class ListUsers
    {
        public static Task<string> RunAsync(FaxCoreClient client, SampleConfig config)
        {
            return client.PostAsync("/api/users/list", new { domain = config.Domain });
        }
    }
}

File: src/javascript/administration/user/list-users.mjs
import { FaxCoreClient, printResult } from '../../common/faxcoreClient.mjs';

const client = new FaxCoreClient();

const result = await client.post('/api/users/list', {
  domain: process.env.FAXCORE_DOMAIN || ''
});

printResult('User list', result);
